packages/fetchai/skills/simple_aggregation/strategy.py

```diff
@@ -179,7 +181,12 @@ def aggregate_observations(self) -> None:
             self.context.logger.info("No observations to aggregate")
             return
         self._aggregation = self._aggregate(values)
-        self.context.shared_state["aggregation"] = self._aggregation
+        self.context.shared_state["aggregation"] = {
```

Comment:
In the shared state the keys themselves should be configurable. Otherwise if two skills define the same key you have a problem if its hard coded.